src/test/java/com/whatsapp/api/impl/WhatsappBusinessCloudApiTest.java

```diff
@@ -1123,6 +1124,31 @@ void testLocationMessage() throws IOException, URISyntaxException, InterruptedEx
         Assertions.assertEquals("wamid.gBGGSFcCNEOPAgkO_KJ55r4w_ww", response.messages().get(0).id());
     }
 
+    @Test
+    void testMarkAsReadMessage() throws IOException, URISyntaxException, InterruptedException, JSONException {
+    	
+    	String responseBody = """
+				                {
+				                "success": true
+				        		}
+				              """;
+        mockWebServer.enqueue(new MockResponse().setResponseCode(200).setBody(responseBody));
+
+        var expectedJson = fromResource(EXPECTED_FOLDER + "expectedMessage17.json");
+
+        var message = new ReadMessage("123456");
+
+        var response = whatsappBusinessCloudApi.markMessageAsRead(PHONE_NUMBER_ID, message);
+
+        RecordedRequest recordedRequest = mockWebServer.takeRequest();
+        Assertions.assertEquals("POST", recordedRequest.getMethod());
+        Assertions.assertEquals("/" + API_VERSION + "/" + PHONE_NUMBER_ID + "/messages", recordedRequest.getPath());
+
+        JSONAssert.assertEquals(expectedJson, recordedRequest.getBody().readUtf8(), JSONCompareMode.STRICT);
+
+        Assertions.assertEquals(true, response.success());
```

Comment:
`Assertions.assertEquals(true, response.success())` can be simplified to `assertTrue()`